core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.spec.ts

```diff
@@ -38,8 +38,9 @@ class MockExpressServerLogger implements Partial<ExpressServerLogger> {
  * 3. Examine renders property for the renders
  */
 class TestEngineRunner {
-  /** Accumulates html output for engine runs */
-  renders: string[] = [];
+  // SPIKE TODO: rename to 'responses'
+  /** Accumulates responses produced by engine runs (either successful html or Error) */
+  renders: (string | Error)[] = [];
```

Comment:
`minor`: Based on the usage and the new type of this field, I vote for renaming it to `responses` as it is proposed in the comment above.